lib/rails/dom/testing/assertions/selector_assertions.rb

```diff
@@ -164,7 +164,33 @@ def assert_dom(*args, &block)
             @selected ||= nil
 
             selector = HTMLSelector.new(args, @selected) { nodeset document_root_element }
+            dom_assertions(selector, &block)
+          end
+          alias_method :assert_select, :assert_dom
+
+          # The negated form of +assert_dom+.
+          #
+          # === Equality Tests
+          #
+          # Supports the same equality tests as +assert_dom+ except for:
+          # * <tt>true</tt>
+          # * <tt>false</tt>
+          # * <tt>Integer</tt>
+          # * <tt>Range</tt>
+          # * <tt>:count</tt>
+          # * <tt>:minimum</tt>
+          # * <tt>:maximum</tt>
+          def assert_not_dom(*args, &block)
+            @selected ||= nil
 
+            selector = HTMLSelector.new(args, @selected, refute: true) { nodeset document_root_element }
+            dom_assertions(selector, &block)
+          end
+          alias_method :refute_dom, :assert_not_dom
+          alias_method :assert_not_select, :assert_not_dom
+          alias_method :refute_select, :assert_not_select
```

Comment:
```suggestion
          alias_method :refute_select, :assert_not_dom
```

I think it's more readable if we're consistent about the alias target

Reply:
Updated [here](https://github.com/rails/rails-dom-testing/compare/b7b3e3a9af76eaec080aab2eee6aa661ed7f56d8..30729665853bb575199a6c66e1d1bb4b5694eb14).